minard/templates/layout.html

```diff
@@ -89,9 +89,10 @@
                                 <li> <a href="https://snopluspmts.physics.berkeley.edu/">Overview</a></li>
                                 {{ nav_link('eca', 'ECA') }}
                                 {{ nav_link('pcatellie', 'PCA Tellie') }}
-				{{ nav_link('noise', 'PMT Noise') }}
+                                {{ nav_link('pcatellie_processing', 'PCA Tellie Processing') }}
```

Comment:
I think it could (with some additional data ported) but I'm not in position to decide this anymore, you would have to ask the Calibration/TELLIE group.

Reply:
OK I'll leave the current page active for now.

Reply:
It might be worth distinguishing the naming a little to make a clear which one should be used.

Reply:
There is the `Processing` bit added on (this tab is really related to the processing of the data), but if you think some other naming would be better I'm happy to rename.